### PR TITLE
Undefined to jest

### PR DESCRIPTION
**Pull request by Madhankumarr**
Jest test migration for "src/test/mochitest/test_reps_undefined.html"

### CODE REVIEW

File: packages/devtools-reps/package.json [hunk 1]
```diff
@@ -36,10 +36,12 @@
     "eslint": "^3.12.0",
     "eslint-plugin-mozilla": "0.2.3",
     "fs-extra": "^2.0.0",
+    "jest": "^19.0.2",
     "jest-cli": "^20.0.4",
     "react": "=15.3.2",
+    "react-addons-test-utils": "=15.3.2",
     "react-dom": "=15.3.2",
-    "react-addons-test-utils": "=15.3.2"
+    "yarn": "^0.24.5"
```
Comment (jasonLaster) [inline packages/devtools-reps/package.json]: hmm, maybe this should be defined in the root package.json

File: packages/devtools-reps/src/reps/tests/undefined.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    const renderedComponent = shallow(Rep({
+      object: stub
+    }));
+    expect(renderedComponent.hasClass('objectBox objectBox-undefined')).toEqual(true);
+  });
+
+});
+
+
+
```
Comment (jasonLaster) [inline packages/devtools-reps/src/reps/tests/undefined.js]: couple extra lines

File: packages/devtools-reps/package.json (hunk 1 above)
Comment (nchevobbe) [inline packages/devtools-reps/package.json]: Yeah, for now @Madhankumarr , can you revert this file and update your PR ?

**Comment by Madhankumarr** [comment]
@nchevobbe  fixed the errors. Kindly check.

**Comment by nchevobbe** [comment]
Thanks @Madhankumarr ! There were some ESLint failures, which I fixed. We can merge this 👍

**Pull request merged**